.gitignore

```diff
@@ -5,3 +5,6 @@ examples/factors/factors
 examples/onlineboutique/onlineboutique
 examples/hello/hello
 website/public/
+
+.idea/
```

Comment:
Can you please remove this? We don't allow adding things in .gitignore that only makes sense on your local machine.

Reply:
This is the approach I use to ignore editor files: https://gist.github.com/subfuzion/db7f57fff2fb6998a16c (another reference: [github docs](https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer)).

It's a great approach because you don't have to ignore these files in every git repository. You set it up once for each machine, and it will ignore the files in all git repositories.

Reply:
Thank you for your suggestion, I will make the necessary changes and resubmit.